Description: Use proper log dir.
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ntopng/src/Prefs.cpp
===================================================================
--- ntopng.orig/src/Prefs.cpp
+++ ntopng/src/Prefs.cpp
@@ -789,7 +789,7 @@ int Prefs::checkOptions() {
       char path[MAX_PATH];
 
       ntop_mkdir(data_dir, 0777);
-      snprintf(path, sizeof(path), "%s/ntopng.log", ntop->get_working_dir() /* "C:\\Windows\\Temp" */);
+      snprintf(path, sizeof(path), "%s/ntopng.log", "/var/log/ntopng");
       ntop->fixPath(path);
       logFd = fopen(path, "w");
       if(logFd)
