Description: Use $(CURDIR) instead of ${PWD} during build.
Author: Ludovico Cavedon <cavedon@debian.org>

Index: ntopng/Makefile.in
===================================================================
--- ntopng.orig/Makefile.in
+++ ntopng/Makefile.in
@@ -22,7 +22,7 @@ endif
 ######
 LIBPCAP=-lpcap
 ######
-MONGOOSE_HOME=${PWD}/third-party/mongoose
+MONGOOSE_HOME=$(CURDIR)/third-party/mongoose
 MONGOOSE_INC=-I$(MONGOOSE_HOME)
 ######
 HAS_LUAJIT=$(shell pkg-config --exists luajit; echo $$?)
@@ -30,12 +30,12 @@ ifeq ($(HAS_LUAJIT), 0)
 	LUAJIT_INC = $(shell pkg-config --cflags luajit)
 	LUAJIT_LIB = $(shell pkg-config --libs luajit)
 else
-	LUAJIT_HOME=${PWD}/third-party/LuaJIT-2.0.3
+	LUAJIT_HOME=$(CURDIR)/third-party/LuaJIT-2.0.3
 	LUAJIT_INC=-I$(LUAJIT_HOME)/src
 	LUAJIT_LIB=$(LUAJIT_HOME)/src/libluajit.a
 endif
 ######
-LIBRRDTOOL_HOME=${PWD}/third-party/rrdtool-1.4.8
+LIBRRDTOOL_HOME=$(CURDIR)/third-party/rrdtool-1.4.8
 HAS_LIBRRDTOOL=$(shell pkg-config --exists librrd; echo $$?)
 ifeq ($(HAS_LIBRRDTOOL), 0)
 	LIBRRDTOOL_INC = $(shell pkg-config --cflags librrd)
@@ -49,14 +49,14 @@ else
 	endif
 endif
 ######
-HTTPCLIENT_INC=${PWD}/third-party/http-client-c/src/
+HTTPCLIENT_INC=$(CURDIR)/third-party/http-client-c/src/
 ######
 HAS_JSON=$(shell pkg-config --exists json-c; echo $$?)
 ifeq ($(HAS_JSON), 0)
 	JSON_INC = $(shell pkg-config --cflags json-c)
 	JSON_LIB = $(shell pkg-config --libs json-c)
 else
-	JSON_HOME=${PWD}/third-party/json-c
+	JSON_HOME=$(CURDIR)/third-party/json-c
 	JSON_INC=-I$(JSON_HOME)
 	JSON_LIB=$(JSON_HOME)/.libs/libjson-c.a
 endif
@@ -71,14 +71,14 @@ ifeq ($(HAS_ZEROMQ), 0)
 		ZEROMQ_LIB = $(ZMQ_STATIC)
 	endif
 else
-	ZEROMQ_HOME=${PWD}/third-party/zeromq-4.1.3
+	ZEROMQ_HOME=$(CURDIR)/third-party/zeromq-4.1.3
 	ZEROMQ_INC=-I$(ZEROMQ_HOME)/include
 	ZEROMQ_LIB=$(ZEROMQ_HOME)/.libs/libzmq.a
 endif
 ######
 TARGET = ntopng
 LIBS = $(NDPI_LIB) $(LIBPCAP) $(LUAJIT_LIB) $(LIBRRDTOOL_LIB) $(ZEROMQ_LIB) $(JSON_LIB) @HIREDIS_LIB@ @SQLITE_LIB@ @MYSQL_LIB@ @LINK_OPTS@ @GEOIP_LIB@ @LDFLAGS@ @PRO_LIBS@ -lm -lpthread
-CPPFLAGS = -g @CFLAGS@ @HIREDIS_INC@ $(MONGOOSE_INC) $(JSON_INC) $(NDPI_INC) $(LUAJIT_INC) $(LIBRRDTOOL_INC) $(ZEROMQ_INC) @MYSQL_INC@ @CPPFLAGS@ -I$(HTTPCLIENT_INC) @PRO_INC@ -DDATA_DIR='"$(datadir)"' -I${PWD}/third-party/libgeohash -I${PWD}/third-party/patricia # -D_GLIBCXX_DEBUG
+CPPFLAGS = -g @CFLAGS@ @HIREDIS_INC@ $(MONGOOSE_INC) $(JSON_INC) $(NDPI_INC) $(LUAJIT_INC) $(LIBRRDTOOL_INC) $(ZEROMQ_INC) @MYSQL_INC@ @CPPFLAGS@ -I$(HTTPCLIENT_INC) @PRO_INC@ -DDATA_DIR='"$(datadir)"' -I$(CURDIR)/third-party/libgeohash -I$(CURDIR)/third-party/patricia # -D_GLIBCXX_DEBUG
 ######
 # ntopng-1.0_1234.x86_64.rpm
 PLATFORM = `uname -p`
